handout/features/spread_and_rest.md

```diff
@@ -1,14 +1,14 @@
-# Operators Spread and Rest
+# Spread Syntax (Spread Element) and Rest parameters
 
-A Spread operator allows in-place expansion of an expression for the following cases:
+A Spread syntax allows in-place expansion of an expression for the following cases:
```

Comment:
@ashwin-sureshkumar showed me the [MDN reference page](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Spread_operator) and they claim that to be a syntax so I agreed to change the name. Interestingly, the url for that page has at the end "Spread_operator". Maybe they left the url for SEO reasons.